src/app/reviewer/[orgSlug]/study/[studyId]/review/test-image-checkbox.tsx

```diff
@@ -0,0 +1,36 @@
+'use client'
+
+import { useQuery } from '@tanstack/react-query'
```

Comment:
need to import this from '@/common' in order to get our wrapped version with error handling, I've just added a eslint rule to enforce this